common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -553,6 +553,11 @@ function (HTML5Video, Resizer) {
         // `duration`. In this case, slider doesn't reach the end point of
         // timeline.
         this.videoPlayer.updatePlayTime(time);
+
+        // Emit 'pause_video' event when a video ends if Player is of Youtube
+        if (this.isYoutubeType()) {
```

Comment:
We are loading the Youtube API if Youtube is available and the API is triggering only ```stop_video```  event when a video ends. This API is not in our code base. So we need to trigger ```pause_video``` event in our code base because youtube API is not triggering that event.


When youtube is not available then we load HTML5 Player and its code is available in our code base & its triggering both events.